site-src/guides/index.md

```diff
@@ -56,55 +55,126 @@ This quickstart guide is intended for engineers familiar with k8s and model serv
-   kubectl rollout restart deployment envoy-gateway -n envoy-gateway-system
-   ```
-   Additionally, if you would like to enable the admin interface, you can uncomment the admin lines and run this again.
+   Choose one of the following options to deploy an Inference Gateway.
 
-### Deploy Gateway
+=== "GKE"
 
-   ```bash
-   kubectl apply -f https://github.com/kubernetes-sigs/gateway-api-inference-extension/raw/main/config/manifests/gateway/gateway.yaml
-   ```
-   > **_NOTE:_** This file couples together the gateway infra and the HTTPRoute infra for a convenient, quick startup. Creating additional/different InferencePools on the same gateway will require an additional set of: `Backend`, `HTTPRoute`, the resources included in the `./config/manifests/gateway/ext-proc.yaml` file, and an additional `./config/manifests/gateway/patch_policy.yaml` file. ***Should you choose to experiment, familiarity with xDS and Envoy are very useful.***
+      1. Enable the Gateway API
 
-   Confirm that the Gateway was assigned an IP address and reports a `Programmed=True` status:
-   ```bash
-   $ kubectl get gateway inference-gateway
-   NAME                CLASS               ADDRESS         PROGRAMMED   AGE
-   inference-gateway   inference-gateway   <MY_ADDRESS>    True         22s
-   ```
-### Deploy the InferencePool and Extension
+         ```bash
+         gcloud container clusters update <CLUSTER_NAME> \
+             --location=<CLUSTER_LOCATION> \
+             --gateway-api=standard
+         ```
 
-   ```bash
-   kubectl apply -f https://github.com/kubernetes-sigs/gateway-api-inference-extension/raw/main/config/manifests/inferencepool.yaml
-   ```
-### Deploy Envoy Gateway Custom Policies
+      1. Create the proxy-only subnet
+      
+         A proxy-only subnet provides a set of IP addresses that Google uses to run Envoy proxies on your behalf. 
+         ```
+         gcloud compute networks subnets create proxy-only-subnet \
+             --purpose=REGIONAL_MANAGED_PROXY \
+             --role=ACTIVE \
+             --region=<REGION> \
+             --network=<VPC_NETWORK_NAME> \
+             --range=<CIDR_RANGE>
+         ```
+
+      1. Deploy Gateway, HTTPRoute and HealthCheckPolicy resources
+
+         ```bash
+         kubectl apply -f https://github.com/kubernetes-sigs/gateway-api-inference-extension/raw/main/config/manifests/gateway/gke/resources.yaml
+         ```
+
+         Confirm that the Gateway was assigned an IP address and reports a `Programmed=True` status:
+         ```bash
+         $ kubectl get gateway inference-gateway
+         NAME                CLASS               ADDRESS         PROGRAMMED   AGE
+         inference-gateway   inference-gateway   <MY_ADDRESS>    True         22s
+         ```
+
+=== "Istio"
+
+      1. Install Istio
+      
+      Follow the Istio installation guide https://istio.io/latest/docs/setup/install/
```

Comment:
note - this will need to be slightly different as the release to be from a branch, and not master. will update once it is released